model/events/parse.go

```diff
@@ -31,7 +32,7 @@ func ParseEvent(eventType flow.EventType) (*ParsedEvent, error) {
 	parts := strings.Split(string(eventType), ".")
 
 	switch parts[0] {
-	case "flow":
+	case "flow", types.EVMLocationPrefix:
```

Comment:
the comment should also be updated.

Reply:
Good point :+1: I missed that.
Updated in https://github.com/onflow/flow-go/pull/5262/commits/dfa6eedaebe634cb05f5f3e8ecc57066e44dc657